programs/drift/src/math/amm_spread.rs

```diff
@@ -295,6 +295,30 @@ pub fn calculate_spread_revenue_retreat_amount(
     Ok(revenue_retreat_amount)
 }
 
+pub fn calculate_max_target_spread(
+    reserve_price: u64,
+    last_oracle_reserve_price_spread_pct: i64,
+    last_oracle_conf_pct: u64,
+    mark_std: u64,
+    oracle_std: u64,
+    max_spread: u32,
+) -> DriftResult<u64> {
+    let max_spread_baseline = last_oracle_reserve_price_spread_pct.unsigned_abs().max(
+        last_oracle_conf_pct
+            .safe_mul(2)?
+            .max(
+                mark_std
+                    .max(oracle_std)
```

Comment:
aren't these percentages already? why do you need to divide by reserve price?

Reply:
they arent, they are in price space